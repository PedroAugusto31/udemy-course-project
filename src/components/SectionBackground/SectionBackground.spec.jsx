import { screen } from "@testing-library/react";
import { SectionBackground } from ".";
import { renderTheme } from "../../utils/renderTheme";

describe("<SectionBackground />", () => {
  it("Should do something", () => {
    renderTheme(<SectionBackground>Children</SectionBackground>);
    expect(screen.getByText("Children")).toBeInTheDocument();
  });
});
