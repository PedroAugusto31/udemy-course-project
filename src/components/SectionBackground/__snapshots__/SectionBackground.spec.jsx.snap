// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground /> Should render with correct background dark 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      Children
    </div>
  </div>
</div>
`;
